### ISSUE TITLE
Make the web interface more accessible

### ISSUE THREAD

**Issue by bpj** [metacpan/metacpan-web#2195]
Please don't override zoom in the web interface.

Is there any deep reason zooming is disabled in the web interface? It's not really very user friendly for people with bad eyesight using handhelds! Personally I just learned how to override it in Chrome but still... I'm certainly old school but I firmly believe in leaving the choice of such things as zoom, text size and font to the user or at least making it easily configurable (preferably in your user settings) since it's an accessibility issue. Myself I have very bad sight on one eye and have become increasingly oversighted rather than shortsighted on the remaining eye. A multi-disabled Perl hacker may be a rare thing but I'm probably not totally alone!

**Comment by jgamble** [comment]
I just checked in Firefox, and using Cntrl-+ and Cntrl-- (control plus and control minus) worked just fine.

Are you only using Chrome, or did you test on other browsers?

**Comment by haarg** [comment]
The problem is specifically on mobile, where the viewport metadata prevents the browser from zooming in.  I'm not sure why that was specified.  It doesn't seem useful.

**Comment by djzort** [comment]
Accessibility might a great idea for a grant.

**Comment by sandbergja** [comment]
I believe that commit e90b2118858622cb5807e93421fcc267ecc6abf4 resolved the specific issue mentioned (unable to zoom on mobile devices).  But there are plenty of other accessibility issues, including several that can be easily detected using a browser extension like WAVE or Axe Devtools, or with Lighthouse in Chromium-based browsers.  Would it make sense to close this particular issue, and open specific issues for the remaining accessibility problems?

**Comment by oalders** [comment]
> Would it make sense to close this particular issue, and open specific issues for the remaining accessibility problems?

Yes, I think that would make it easier to track what's going on. 😄

**Comment by oalders** [comment]
We now have some other accessibility issues, so I'll close this one.
